manifests/07-operator-ibm-cloud-managed.yaml

```diff
@@ -37,6 +37,8 @@ spec:
           value: docker.io/openshift/origin-docker-registry:latest
         - name: IMAGE_PRUNER
           value: quay.io/openshift/origin-cli:v4.0
+        - name: AZURE_ENVIRONMENT_FILEPATH
+          value: /tmp/azurestackcloud.json
```

Comment:
Is this some kinda default location for this? I could not see where this file is used, IIUC the endpoints are copied into it but I could not find where it is used.

Reply:
azure sdk uses AZURE_ENVIRONMENT_FILEPATH to find its config

Reply:
Oh yes, of course. Shame on me.